README.md

```diff
@@ -1,2354 +1,184 @@
+  open /Applications/Python\ 3.12/Update\ Shell\ Profile.command
+  ```
+- A Python app. Use your favorite IDE, or run from the command line
+- An [API key and secret](https://support.bitvavo.com/hc/en-us/articles/4405059841809) associated with your Bitvavo account
+
+  You control the actions your app can do using the rights you assign to the API key. 
+  Possible rights are:
+  + **View**: retrieve information about your balance, account, deposit and withdrawals
+  + **Trade**: place, update, view and cancel orders
+  + **Withdraw**: withdraw funds
+
+    Best practice is to not grant this privilege, withdrawals using the API do not require 2FA and e-mail confirmation.
+
+## Get started
+
+Want to quickly make a trading app? Here you go: 
+
+1. **Install Bitvavo SDK for Python**  
+
+    In your Python app, add [Bitvavo SDK for Python](https://github.com/bitvavo/python-bitvavo-api) from [pypi.org](https://pypi.org/project/python-bitvavo-api/):
+    ```terminal
+    python -m pip install python_bitvavo_api
+    ```
+1. **Create a simple Bitvavo implementation**
+
+    Add the following code to a new file in your app:
+
+    ```python
+    from python_bitvavo_api.bitvavo import Bitvavo
+    import json
+    import time
+    
+    # Use this class to connect to Bitvavo and make your first calls.
+    # Add trading strategies to implement your business logic.
+    class bitvavo_implementation:
+        api_key = "<Replace with your your API key from Bitvavo Dashboard>"
+        api_secret = "<Replace with your API secret from Bitvavo Dashboard>"
+        bitvavo_engine = None
+        bitvavo_socket = None
+    
+        # Connect securely to Bitvavo, create the WebSocket and error callbacks.
+        def __init__(self):
+            self.bitvavo_engine = Bitvavo({
+                'APIKEY': self.api_key,
+                'APISECRET': self.api_secret
+            })
+            self.bitvavo_socket = self.bitvavo_engine.newWebsocket()
+            self.bitvavo_socket.setErrorCallback(self.error_callback)
+    
+        # Handle errors.
+        def error_callback(self, error):
+            print("Errors:", json.dumps(error, indent=2))
+    
+        # Retrieve the data you need from Bitvavo in order to implement your
+        # trading logic. Use multiple workflows to return data to your
+        # callbacks.
+        def a_trading_strategy(self):
+            self.bitvavo_socket.ticker24h({}, self.a_trading_strategy_callback)
+    
+        # In your app you analyse data returned by the trading strategy, then make
+        # calls to Bitvavo to respond to market conditions.
+        def a_trading_strategy_callback(self, response):
+            print("All the latest trades:", json.dumps(response, indent=2))
+            # Iterate through the
+            for market in response:
+                print("Iterate through markets:", market["market"] )
+                match market["market"]:
+                   case "A market":
```

Comment:
:art: How about using a concrete market here?
```suggestion
                   case "BTC-EUR":
```
Instead of using an existing one, I otherwise also use sth like `"ABC-EUR"` or similar

Reply:
Done